iceoryx_examples/waitset/README.md

```diff
@@ -358,7 +361,7 @@ After that we require a `cyclicTrigger` to trigger our
 triggerId `0` and the callback `SomeClass::cyclicRun`
 ```cpp
 iox::popo::UserTrigger cyclicTrigger;
-cyclicTrigger.attachTo(waitset, 0, SomeClass::cyclicRun);
+cyclicTrigger.attachEvent(waitset, SomeClass::cyclicRun);
```

Comment:
The UserTrigger method is called `attachTo`, or? I fear the naming discussion is not yet over. The master of waitset mixed up things themselves ;-)

Reply:
fixed by redesign